x/txfees/keeper/feedecorator.go

```diff
@@ -59,7 +59,6 @@ func (mfd MempoolFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate b
 		}
 	}
 
-	// UNFORKINGTODO C: Added this logic so that we can create gentx's without having to pay a fee.
```

Comment:
We used to have custom logic for IsGenesis, it never existed in the sdk. It was technically more right, but I don't see an instance where someone could modify the block height to be zero in a non genesis scenario, so this seems safe to me.